src/client/datascience/jupyter/jupyterNotebook.ts

```diff
@@ -329,11 +329,13 @@ export class JupyterNotebookBase implements INotebook {
                 }
             } else {
                 this.initializedMatplotlib = false;
+                //const configInit =
+                //(!settings || settings.enablePlotViewer) &&
+                //!isResourceNativeNotebook(this._resource, this.vscNotebook, this.fs)
+                //? CodeSnippets.ConfigSvg
+                //: CodeSnippets.ConfigPng;
                 const configInit =
-                    (!settings || settings.enablePlotViewer) &&
-                    !isResourceNativeNotebook(this._resource, this.vscNotebook, this.fs)
-                        ? CodeSnippets.ConfigSvg
-                        : CodeSnippets.ConfigPng;
+                    settings && settings.enablePlotViewer ? CodeSnippets.ConfigSvg : CodeSnippets.ConfigPng;
```

Comment:
Removed the !isResourceNativeNotebook as Native notebooks can now use the setting to opt into getting the SVGs if they want. 

This code also doesn't look to have worked before. The check for isNativeNotebook was here, but not in the kernel.ts version. So this init would run first and set to [png] then the init in the native kernel.ts code would run and set to [png, svg] so native notebooks were always getting the SVGs unless the setting was off.